tests/unit/dom.ts

```diff
@@ -237,5 +237,169 @@ registerSuite({
+							dom.containsClass(null, 'test');
+						});
+					},
+
+					'invalid class'() {
+						assert.throws(function () {
+							dom.containsClass(targetElement, 'te est');
+						});
+					},
+
+					'null class'() {
+						assert.doesNotThrow(function () {
+							dom.containsClass(targetElement, null);
+						});
+					}
+				},
+
+				removeClass: {
+					afterEach() {
+						targetElement.className = '';
+					},
+
+					'remove single class'() {
+						dom.addClass(targetElement, 'test');
+						assert.include(targetElement.className, 'test');
+						dom.removeClass(targetElement, 'test');
+						assert.notInclude(targetElement.className, 'test');
+					},
+
+					'remove multiple classes'() {
+						dom.addClass(targetElement, 'test1', 'test2');
+						assert.include(targetElement.className, 'test1');
+						assert.include(targetElement.className, 'test2');
+						dom.removeClass(targetElement, 'test1', 'test2');
+						assert.notInclude(targetElement.className, 'test1');
+						assert.notInclude(targetElement.className, 'test2');
+					},
+
+					'null node'() {
+						assert.doesNotThrow(function () {
+							dom.removeClass(null, 'test');
+						});
+					},
+
+					'invalid class'() {
+						assert.throws(function () {
+							dom.removeClass(targetElement, 'te est');
+						});
+					},
+
+					'null class'() {
+						assert.doesNotThrow(function () {
+							dom.removeClass(targetElement);
+						});
+					},
+
+					'remove nonexistent class'() {
+						assert.doesNotThrow(function () {
+							dom.removeClass(targetElement, 'random');
+						});
+					},
+
+					'remove mutiple nonexistent classes'() {
+						assert.doesNotThrow(function () {
+							dom.removeClass(targetElement, 'random', 'random1');
+						});
+					}
+				},
+
+				toggleClass: {
+					'toggling existing class removes it'() {
+						dom.addClass(targetElement, 'test');
+						assert.include(targetElement.className, 'test');
+						dom.toggleClass(targetElement, 'test');
+						assert.notInclude(targetElement.className, 'test');
+					},
+
+					'toggling nonexistent class adds it'() {
+						dom.toggleClass(targetElement, 'test');
+						assert.include(targetElement.className, 'test');
+					},
+
+					'toggling class using force = true'() {
+						dom.toggleClass(targetElement, 'test', true);
+						assert.include(targetElement.className, 'test');
+						dom.toggleClass(targetElement, 'test', true);
+						assert.include(targetElement.className, 'test');
+					},
+
+					'toggling class using force = false'() {
+						dom.toggleClass(targetElement, 'test', false);
+						assert.notInclude(targetElement.className, 'test');
+						dom.addClass(targetElement, 'test');
+						dom.toggleClass(targetElement, 'test', false);
+						assert.notInclude(targetElement.className, 'test');
+					}
+				}
+			};
+		}
+
+		return {
+			HTMLElement: cssSuite(document.createElement('div')),
+			SVGElement: cssSuite(document.createElement('svg'))
```

Comment:
I thought most of SVG required you to use things like createElementNS, and provide the namespace as a parameter... but maybe that's been fixed/simplified. That said, I was thinking this was for all DOM tests, not just the class manipulation subset. So I guess I'm more concerned about fromString and place being correct for an SVG environment, of the things we currently include. I also thought for getting and setting classes in SVG you had to use getAttribute/setAttribute rather than relying on element.className? Anyway, I guess my overall point is to make sure we are reviewing this with SVG support in mind.